### ISSUE TITLE
ESC #1 on crazybeeF4DX missing

### ISSUE THREAD

**Issue by CosmicChicken777** [jflight-public/jesc-configurator#83]
I'm attached the log - tried flashing JESC and now ESC missing.  BL Heli won't see it either - followed instructions on JESC troubleshooting
[Log.txt](https://github.com/jflight-public/jesc-configurator/files/5899099/Log.txt)

**Comment by CosmicChicken777** [comment]
@joelucid do you have any ideas?  Brand new board.  Trying to get JESC on it, I already paid the license fee, but then it lost the ESC #1 in the flashing process.

**Comment by joelucid** [comment]
The log shows esc 1 missing even when you read the settings the first time. So it seems that it went missing during a flash attempt using blheli or one of its derivatives? JESC configurator has many flash stability related bug fixes and features which other blheli_s derivatives lack.

**Comment by CosmicChicken777** [comment]
@joelucid I used a different computer for the original flash.  If I plugged it into the original computer and save the log, would that help you?

**Comment by joelucid** [comment]
Yeah with the other log I can try to see what kind of failure occurred.

**Comment by CosmicChicken777** [comment]
@joelucid here is the log from the other computer where the I first tried flashing with JESC.  Thanks!
[larva-jesc-Log3.txt](https://github.com/jflight-public/jesc-configurator/files/5939325/larva-jesc-Log3.txt)

**Comment by CosmicChicken777** [comment]
> @joelucid here is the log from the other computer where the I first tried flashing with JESC. Thanks!
> [larva-jesc-Log3.txt](https://github.com/jflight-public/jesc-configurator/files/5939325/larva-jesc-Log3.txt)

@joelucid  Any luck looking at the log?  Is my Crazybeef4DX ruined?

**Comment by CosmicChicken777** [comment]
@joelucid Hello?  You able to look at the previous log?
